app/viewmodels/BeenOnStatutoryLeaveHelper.scala

```diff
@@ -63,9 +65,41 @@ class BeenOnStatutoryLeaveHelper extends EmployeeTypeUtil with KeyDatesUtil {
         None
     }
 
-  def type3And4BoundaryEnd()(implicit request: DataRequest[_], messages: Messages): String = {
+  def type3And4BoundaryEndMessage()(implicit request: DataRequest[_], messages: Messages): String = {
     val dayBeforeFirstFurlough = firstFurloughDate().minusDays(1)
     Logger.debug(s"[BeenOnStatutoryLeaveHelper][type3And4BoundaryEnd] dayBeforeFirstFurlough: $dayBeforeFirstFurlough")
     dateToString(earliestOf(apr5th2020, dayBeforeFirstFurlough))
   }
+
+  def boundaryStartDate()(implicit request: DataRequest[_], appConfig: FrontendAppConfig, messages: Messages): LocalDate = {
```

Comment:
Sorry Will this is my bad this should go into it's own helper class for your page/controller